src/components/Footer/Footer.js

```diff
@@ -56,7 +56,7 @@ Footer.propTypes = {
 Footer.defaultProps = {
   noLinks: false,
   noSocial: false,
-  noEndpoint: true,
+  forceFallback: true,
```

Comment:
Sorry man, I missed this- probably best that this defaults to FALSE; we don't want to make a sweeping change like this to existing products using the footer.

Reply:
Done